core/sms_server_v2.py

```
@@ -0,0 +1,592 @@
+        power_down_count = db.query(PowerDownStore).count()
+        if power_down_count > 0:
+            redis_client.restore_from_power_down_store(db)
+            logger.info(f"Restored {power_down_count} entries from power_down_store")
+    
+    # 5. Load blacklist from database
+    from core.database import get_db_context
+    from core.models import BlacklistMobile
+    
+    with get_db_context() as db:
+        blacklist = db.query(BlacklistMobile).filter(
+            BlacklistMobile.is_active == True
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `BlacklistMobile.is_active:` for truth checks

Replace with `BlacklistMobile.is_active`

(E712)